infra/gcp/ensure-conformance-storage.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to create a new "conformance" bucket in GCS.
+#
+# Each conformance bucket exists in its own GCP project, and is writable by a
+# dedicated googlegroup.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+. "${SCRIPT_DIR}/lib.sh"
+
+function usage() {
+    echo "usage: $0 [repo...]" > /dev/stderr
+    echo "example:" > /dev/stderr
+    echo "  $0 # do all staging repos" > /dev/stderr
+    echo "  $0 coredns # just do one" > /dev/stderr
+    echo > /dev/stderr
+}
+
+# NB: Please keep this sorted.
+CONFORMANCE_PROJECTS=(
+    capi-openstack
+)
+if [ $# = 0 ]; then
+    # default to all staging projects
+    set -- "${CONFORMANCE_PROJECTS[@]}"
+fi
+
+for REPO; do
```

Comment:
fyi, I'm not sure how the REPO variable is actually filled. I just replicated the behavior of: https://github.com/kubernetes/k8s.io/blob/master/infra/gcp/ensure-staging-storage.sh#L67

Reply:
it iterates "$@", which is either set from commandline or above in the `set --`

Reply:
Ah thx didn't know that